.github/actions/deploy/action.yaml

```diff
@@ -3,7 +3,7 @@ description: Deploy on AWS S3 Bucket
 
 inputs:
   environment:
```

Comment:
**question (non-blocking)**: why not change the input from `environment` to `tenant` then?

Reply:
Today we configured the env var on github with Christian for the tenant send test environment. The test has ended, so the env var will need to be modified with the new CDN values (for production). So I'm putting the PR in draft until everything is ready for the production environment.

The input was originally called tenant, then we decided in call to leave it with the name environment